Dockerfile

```diff
@@ -1,5 +1,20 @@
-FROM ubuntu:20.04
+#Add qemu function for supporting multiarch
+ARG IMAGEARCH
+FROM alpine:3.9.2 as qemu
```

Comment:
This is a old version of Alpine Linux. The current GA is 3.12. Why have you used this specific version?

Reply:
The alpine is a tool image which is used for copy qemu binary, but not a base image. The real base image is ubuntu:20.04. So, I choose it at random. If you prefer the latest version, I think it is OK.

Reply:
I'm aware of what a multi-stage Docker image build is. Please update the image tag.